src/charm.py

```diff
@@ -54,14 +56,23 @@ def __init__(self, *args):
         super().__init__(*args)
         self._container = self.unit.get_container(self._name)
         self._stored.set_default(agent_config="")
+        self._metrics_rules_src_path = os.path.join(self.charm_dir, METRICS_RULES_SRC_PATH)
+        self._metrics_rules_path = os.path.join(self.charm_dir, METRICS_RULES_PATH)
```

Comment:
Consider adding:
- utest: forwarded rules remain intact
- itest: forwarded rule files show up when queried for via http api